services/storage/src/simcore_service_storage/dsm.py

```diff
@@ -545,9 +546,14 @@ async def _init_metadata() -> Tuple[int, str]:
                 object_name=object_name,
             )
         )
-        return self.s3_client.create_presigned_put_url(bucket_name, object_name)
+        link = URL(f"s3://{bucket_name}/{object_name}")
```

Comment:
I'd go for pedantic, it works nicely

```suggestion
        link = parse_obj_as(AnyUrl, f"s3://{bucket_name}/{object_name.lstrip('/')}")
```